components/homme/dcmip_tests/dcmip2016_test2_tropical_cyclone/theta-l/namelist-r100.nl

```diff
@@ -1,51 +1,51 @@
 !
-! theta: namelist for dcmip2016 test2: tropical cyclone
```

Comment:
For the changes in *nl files, there are too many formatting changes to evaluate by eye. I am not sure those runs are used for, say, energy cascades or anything else, so i am not sure how important that those files are not being modified (that is, that params in the namelists are not modified). Maybe @mt5555 has an opinion.

Reply:
If you go to the very top of this page, the gear icon has a "hide whitespace" option.  There's still a lot of changes from changing comments to mixed case.